test/unit/test_retry_network.py

```diff
@@ -180,6 +180,7 @@ def test_request_exec():
         "headers": {},
         "data": '{"code": 12345}',
         "token": None,
+        "params": None,
```

Comment:
can we add tests to ensure credentials are not leaked in log? we can use pytest caplog to capture logs